### ISSUE TITLE
Chromedriver version m1 mac

### ISSUE THREAD

**Issue by bbereczky-wecan** [giggio/node-chromedriver#399]
## TLDR
From `106.0.5249.61` a new naming convention came in place: `chromedriver_mac_arm64` (`106.0.5249.61/chromedriver_mac_arm64`)
So please replace the naming convention from `chromedriver_mac64_m1` to `chromedriver_mac_arm64`

## Description

When I try to install chromedriver on my m1 mac I receive this issue.
```

> chromedriver@105.0.1 install /Users/bbereczky/workspace/wecanconsulting/OSNGateway.ComponentTests/node_modules/chromedriver
> node install.js

Your Chrome version is 107.0.5304.110
Finding Chromedriver version.
Chromedriver version is 107.0.5304.62.
Compatible ChromeDriver version is 107.0.5304.62
Current existing ChromeDriver binary is unavailable, proceeding with download and extraction.
Downloading from file:  https://npmmirror.com/mirrors/chromedriver/107.0.5304.62/chromedriver_mac64_m1.zip
Saving to file: /var/folders/s2/hnt6m9y51832wjl6z1mlhm8w0000gn/T/107.0.5304.62/chromedriver/chromedriver_mac64_m1.zip
Error status code: 404
<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>NoSuchKey</Code>
  <Message>The specified key does not exist.</Message>
  <RequestId>6376118F39AA693239595B93</RequestId>
  <HostId>npmmirror.oss-cn-shanghai.aliyuncs.com</HostId>
  <Key>binaries/chromedriver/107.0.5304.62/chromedriver_mac64_m1.zip</Key>
</Error>

ChromeDriver installation failed Error: Error with http(s) request: AxiosError: Request failed with status code 404
    at requestBinary (/Users/bbereczky/workspace/wecanconsulting/OSNGateway.ComponentTests/node_modules/chromedriver/install.js:274:11)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async downloadFile (/Users/bbereczky/workspace/wecanconsulting/OSNGateway.ComponentTests/node_modules/chromedriver/install.js:117:5)
    at async install (/Users/bbereczky/workspace/wecanconsulting/OSNGateway.ComponentTests/node_modules/chromedriver/install.js:66:7)
```

The problem is that the naming convention have been changed in google.
[https://chromedriver.storage.googleapis.com/](https://chromedriver.storage.googleapis.com/)
Here the latest version using the `chromedriver_mac64_m1` naming conveintion is : `106.0.5249.21`
in `106.0.5249.21/chromedriver_mac64_m1.zip`

From `106.0.5249.61` a new naming convention came in place: `chromedriver_mac_arm64` (`106.0.5249.61/chromedriver_mac_arm64`)

So please replace the naming convention from `chromedriver_mac64_m1` to `chromedriver_mac_arm64`

**Comment by bbereczky-wecan** [comment]
It's solved I was using a 105 version instead of 107.
